src/AbstractPolytope.jl

```diff
@@ -128,6 +97,22 @@ function default_polyhedra_backend(P, N)
     error("no default backend for numeric type $N")
 end
 
+# given a polytope P, apply the linear map P to each vertex of P
+# it is assumed that the interface function `vertices_list(P)` is available 
+@inline function _linear_map_vrep(M::AbstractMatrix{N}, P::AbstractPolytope{N}) where {N<:Real}
+    return broadcast(v -> M * v, vertices_list(P)) |> VPolytope{N}
+end
+
+function _linear_map_hrep(M::AbstractMatrix{N}, P::AbstractPolytope{N},
+                          use_inv::Bool) where {N<:Real}
+    constraints = _linear_map_hrep_helper(M, P, use_inv)
+    return dim(P) == 2 ? HPolygon(constraints) : HPolytope(constraints)
```

Comment:
A few lines above the `_linear_map_vrep` always returns a `VPolytope` even in 2D. I find this inconsistent.

Reply:
Yes, to fix the case of polygons and polytopes by writing specific methods for these is the purpose of #1132 and #1133.

Reply:
They are so related to this PR that let's do them on this issue as well.